src/ImportExport.ts

```diff
@@ -173,6 +224,12 @@ export const exportSynergism = async () => {
         : 'Savefile copied to file!';
 }
 
+export const errorGame = async () => {
+    await Alert('sorry. You may be using an older version of your browser, current save data may be incorrect, or some problem may prevent from starting the game\nThis button just deletes the save data, but Synergism will work\nThen import the exported save data have\nCan report the console in error on Discord Synergism channel');
+    await Alert('The next confirmation is to delete the save data\nIf you do not want to delete it, cancel it');
```

Comment:
You should be able to cancel it from here; `resetGame` doesn't have a good cancellation prompt (you have to answer incorrectly which isn't good ux)